packages/dds/tree/src/test/utils.ts

```diff
@@ -1421,3 +1422,17 @@ function normalizeNewFieldContent(
 
 	return cursorForMapTreeField([mapTreeFromCursor(content)]);
 }
+
+/**
+ * Convenience helper for performing a "move" edit where the source and destination field are the same.
+ * @remarks Accepts either an editor or an object which has an editor
```

Comment:
How does the "object which has an editor" part work? I'm not seeing it in the API.

Reply:
Oops - that was leftover from something else - I will remove, thanks.